models/issues/comment.go

```diff
@@ -130,6 +130,8 @@ const (
 	CommentTypePRScheduledToAutoMerge
 	// 35 pr was un scheduled to auto merge when checks succeed
 	CommentTypePRUnScheduledToAutoMerge
+	// 36 tag PR commit
+	CommentTypeTagPRCommit
```

Comment:
why do we need a new comment?, normal tagging does not create a commit

Reply:
@6543 this is an action like a merge, so that it's visible in the timeline that the tag has been set by gitea. Like we also show branch deletes. See also screenshot above where it says "wim added tag xxx to yyy"

Fine for me to remove it though.